server/controllers/api/documents.js

```diff
@@ -0,0 +1,177 @@
+/* eslint import/no-unresolved: 0 */
+import db from '../../models/';
+
+const docAttributes = (doc) => {
+  const attributes = {
+    id: doc.id,
+    title: doc.title,
+    content: doc.content,
+    OwnerId: doc.OwnerId,
```

Comment:
the reason I user the uppercase for it is because it is a foreign key and I wanted it to be unique from other fields. Is that ok?